src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -216,7 +236,7 @@ public Team getTeamByMentorId(Id mentorId) throws AlfredException {
         for (Team t: teams) {
             Optional<Mentor> mentor = t.getMentor();
             if (mentor.isPresent()) {
-                if (mentor.get().getId() == mentorId) {
+                if (mentor.get().getId().equals(mentorId)) {
```

Comment:
hahaah nitpicking, the error thrown should be:
 throw new AlfredRuntimeException("Team with said mentor cannot be found."); 
instead of:
 throw new AlfredRuntimeException("Team with said participant cannot be found.");